### ISSUE TITLE
In the mailbox, the "Sales pending" Letters should just dissapea...

### ISSUE THREAD

**Issue by kaytotes** [Project-Epoch/bugtracker#5362]
> In the mailbox, the "Sales pending" Letters should just dissapear once the payment is delivered

.go xyz -8873.830000 648.600000 96.032700 0 2.308770

**Comment by Bennylavaa** [comment]
I agree these are annoying, it would be better off not getting these at all to be honest.
